float64[] Angles
float64[] Vel
float64[] Torque
